DRAFT_COROLLARY.md

```diff
@@ -277,7 +279,6 @@ explicit constexpr polymorphic(allocator_arg_t, const Allocator& a, U&& u);
 
 D. _Constraints_: `is_base_of_v<T, std::remove_cvref_t<U>>` is `true`.
    `is_copy_constructible_v<remove_cvref_t<U>>` is `true`.
-   `is_same_v<remove_cvref_t<U>, in_place_t>` is `false`.
```

Comment:
Should this have `remove_cvref_t<U>` is not a specialization of `in_place_type_t`? And similarly for the ctor above.

Reply:
`is_base_of_v<T, std::remove_cvref_t<U>>` is `true` should cover us as `in_place_type_t` is not derived from `T` or equal to `T` for any legal choice of `T`.

We can add the constraint if it helps to be explicit.

Reply:
Agree. `is_base_of_v<T, std::remove_cvref_t<U>>` is `true` should have use covered but no harm in being explicit - better errors that way.